app/menus/menu.js

```diff
@@ -17,6 +17,16 @@ const appVersion = app.getVersion();
 
 module.exports = (createWindow, updatePlugins, getLoadedPluginVersions) => {
   const commands = getKeymaps().commands;
+
+  const contextMenuTemplate = () => {
+    const seperator = {type: 'separator'};
+    return editMenu(commands).submenu
+    .concat(seperator, shellMenu(commands, createWindow).submenu);
```

Comment:
you are using it once. no need for `const` there 
simply use {type: 'separator'}